### ISSUE TITLE
Decouvrebitcoin.fr use case : manage paying stream through bootik

### ISSUE THREAD

**Issue by Tirodem** [be-BOP-io-SA/be-BOP#287]
TBD

tl;dr :
- bootik allows teacher to register, provide paying remote courses & take a defined % commission
- teachers can register and make themselves available through an UI form
  - they also 
- available teachers profiles are displayed on the shop like product (based on https://github.com/B2Bitcoin/B2BitcoinBootik/issues/286 )
  - product type : "VISIO" + svg to identify
  - CTA : "Initiate meet"
- when a customer initiates meet through CTA, redirect to a page with Jitsi integration ( https://jitsi.org/api/ )
  - the teacher receives a notification on nostr to accept or decline the meet
    - if decline : end of meet, the customer is notified on the page, nothing is billed
    - if accept :
      - teacher joins customer on 1-1 jitsi meet through bootik page
      - after 2 mn, each minute is billed (through https://satoshis.stream/ -like feature) in 2 ways :
        - a first sat stream between the customer and the teacher, with teacher defined amount
        - a 2nd sat stream between the teacher and the bootik, with teacher defined amount * owner defined commission %
        - once one or another doesn't stream sats anymore, the visio is cut the next minute (both participants are kicked out)

**Comment by Tirodem** [comment]
Those cases will be covered by PipeliNostr or something similar.
